twill-yarn/src/main/java/org/apache/twill/yarn/YarnTwillController.java

```diff
@@ -77,7 +77,7 @@
    */
   YarnTwillController(String appName, RunId runId, ZKClient zkClient,
                       final ApplicationMasterLiveNodeData amLiveNodeData, final YarnAppClient yarnAppClient) {
-    super(runId, zkClient, Collections.<LogHandler>emptyList());
+    super(appName, runId, zkClient, amLiveNodeData.getKafkaZKConnect() != null, Collections.<LogHandler>emptyList());
```

Comment:
Can the constructor of `YarnTwillController` take new input of `logCollectionEnabled` ? I am seeing different ways to check whether to disable log or not